src/WinRT.Runtime/ActivationFactory.cs

```diff
@@ -197,8 +197,27 @@ public static unsafe (ObjectReference<I> obj, int hr) GetActivationFactory<I>(st
 #endif
     static class ActivationFactory
     {
+#if NET
+        /// <summary>
+        /// This provides a hook into activation to hook/mock activation of WinRT types.
+        /// </summary>
+        public static Func<string, Guid, IntPtr> ActivationHandler { get; set; }

```

Comment:
Looks good, is there an easy test case we can add for this just to make sure we don't regress it and that it works.

Reply:
Will add a test.